pkg/job/discovery.go

```diff
@@ -82,6 +82,10 @@ func scrapeDiscoveryJobUsingMetricData(
 		return resources, cw
 	}
 
+	if len(resources) == 0 {
+		logger.Debug("No tagged resources", "region", region, "namespace", job.Type)
+	}
+
```

Comment:
This log line was within the loop below but it makes more sense to be here.

Reply:
I _kinda_ understand the original intent of saying "this metric can't be associated back to resources". But I 100% agree it's better here since it's a fact for the all metrics in the aws namespace not just one or two metrics.